### PR TITLE
Adds SciQLop and PyCDFpp projects and update a bit Speasy entry

### PR DESCRIPTION
**Pull request by jeandet**
SciQLop has quite some red badges but that would be nice if we can advertise it a bit.

### CODE REVIEW

**Comment by sapols** [comment]
Hi, sorry for the super slow response. I've got a backlog of other tasks before I can start this review, and with holidays/conferences it might be a while.

**Comment by sapols** [comment]
Sorry again, I've had a slow start to the year. I'll get to this review in the next week or so. Just wanted to let you know it's still bubbling up my to-do list.

**Comment by jeandet** [comment]
> Sorry again, I've had a slow start to the year. I'll get to this review in the next week or so. Just wanted to let you know it's still bubbling up my to-do list.

No problem 😁.

File: _data/projects.yml [hunk 1]
```diff
@@ -565,3 +566,29 @@
   software_maturity: ["https://img.shields.io/badge/Partially%20met-orange.svg", "Partially met"]
   python3: ["https://img.shields.io/badge/Good-brightgreen.svg", "Good"]
   license: ["https://img.shields.io/badge/Good-brightgreen.svg", "Good"]
+
+- name: PyCDFpp
+  code: https://github.com/SciQLop/CDFpp
+  description: "A fast and easy to use C++ CDF library with Python bindings."
+  docs: https://pycdfpp.readthedocs.io/en/latest/
+  contact: Alexis Jeandet
+  keywords: ["heliophysics", "space physics", "magnetospheric physics", "CDF", "data"]
```
Comment (sapols) [inline _data/projects.yml:575]: Please redo these keywords as they must come from PyHC's [taxonomy](https://github.com/heliophysicsPy/heliophysicsPy.github.io/blob/main/_data/taxonomy.yml). If you feel the taxonomy is missing something, you may add to it in this PR.

File: _data/projects.yml [hunk 2]
```diff
@@ -565,3 +566,29 @@
...
+  software_maturity: ["https://img.shields.io/badge/Good-brightgreen.svg", "Good"]
+  python3: ["https://img.shields.io/badge/Good-brightgreen.svg", "Good"]
+  license: ["https://img.shields.io/badge/Good-brightgreen.svg", "Good"]
+
+- name: SciQLop
+  code: https://github.com/SciQLop/SciQLop
+  description: "SciQLOP (SCIentific Qt application for Learning from Observations of Plasmas) is a powerful and user-friendly tool designed for the visualization and analysis of in-situ space plasma data."
+  logo: https://raw.githubusercontent.com/SciQLop/SciQLop/main/SciQLop/resources/icons/SciQLop.png
+  contact: Alexis Jeandet
+  keywords: ["heliophysics", "space physics", "magnetospheric physics", "Machine-learning", "data"]
```
Comment (sapols) [inline _data/projects.yml:588]: Same comment for these keywords.

**Review by sapols — CHANGES_REQUESTED** [review]
I agree with your self-evaluations. We just can't merge this until the keywords are fixed. Please let me know if you'd like any assistance.

**Comment by sapols** [comment]
Finally got around to reviewing this. I notice there's a merge conflict now though, and that's my bad. I merged another PR before yours that added Hermes-Core to this `projects.yml`. I'll see if I can resolve it myself.

**Comment by sapols** [comment]
Merge conflict resolved. I put the HERMES-Core entry above your new ones.

**Comment by jeandet** [comment]
@sapols I've rebased the commit with the requested modifications, sorry I've missed the guidelines for keywords 😅.

**Review by sapols — APPROVED** [review]
Awesome! No worries, this PR looks good now. Welcome SciQLop and PyCDFpp to PyHC.

**Pull request merged**